utils/helper.js

```diff
@@ -1,6 +1,7 @@
-import startCase from 'lodash.startcase'
+/* eslint-env node */
```

Comment:
Why is this needed?

Reply:
We added two global variables for CommonJS: `require` and `exports`, linter will not understand them if we didn't explicitly say that this is node code.

Reply:
How come it's not needed in all the other Node files?

Reply:
@iAdramelk could you please give some context? should we add it there?

Reply:
@jorgeorpinel But server.js has it, I copied it from there. :)

Reply:
Basically in `.eslitrc.json` we wrote which set of global variables to use`{ env: browser: true } }` then linting files and then we wrote `lint-staged` rule in `package.json` that says which files to lint on commit: `"lint-staged": { "*.{js,jsx}": "eslint" }`. 

This way, all `*.{js,jsx}` files that we try to commit will be linted. Files that are written using `import`/`export` will be linted automatically, but for the node files, we need to add this rule.

There is an inconsistency in our config right now: We lint all files on commit, but only lint `pages` and `src` folders on CI. It can lead to a situation when some old node files still didn't have this rule because we didn't change them in a long time. I think we should also lint all files on CI.

Reply:
Ah sorry Idk why I didn't find that in server.js before 😅 

> There is an inconsistency in our config right now: We lint all files on commit, but only lint pages and src folders on CI. I think we should also lint all files on CI.

We recently noticed some problems with the CI formatting but, only on the `master` branch. Not exactly what you're describing though. We link only staged files on commit:

```yarn
    "format-staged": "pretty-quick --staged --no-restage --bail",
...
      "pre-commit": "yarn format-staged && lint-staged"
...
  "lint-staged": {
    "*.{js,jsx}": "eslint"
```

But anyway, yes I need to review the hook and CI setup. Thanks for the note!